lib/result-model.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Do you actually need all these `use strict`? In es6 every module run in strict mode by default.